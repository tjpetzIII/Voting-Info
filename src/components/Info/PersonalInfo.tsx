import ProfessionalLinks from "./ProfessionalLinks";

const PersonalInfo = () => {
  return (
    <div className="ml-8">
      <img
        src="/images/personalPicture.png"
        alt="Profile Image"
        className="w-40 h-40 pl-5"
      />
      <h2 className="text-2xl">Thomas Petz</h2>
      <ProfessionalLinks />
    </div>
  );
};

export default PersonalInfo;
