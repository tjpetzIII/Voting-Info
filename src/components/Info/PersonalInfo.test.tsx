import { render } from "@testing-library/react";
import PersonalInfo from "./PersonalInfo";

describe("PersonalInfo", () => {
  it("matches the snapshot", () => {
    const { asFragment } = render(<PersonalInfo />);
    expect(asFragment()).toMatchSnapshot();
  });
});
