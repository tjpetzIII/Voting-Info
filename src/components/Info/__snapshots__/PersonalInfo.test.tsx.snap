// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PersonalInfo matches the snapshot 1`] = `
<DocumentFragment>
  <div
    class="ml-8"
  >
    <img
      alt="Profile Image"
      class="w-40 h-40 pl-5"
      src="/images/personalPicture.png"
    />
    <h2
      class="text-2xl"
    >
      Thomas Petz
    </h2>
    <nav
      class="flex items-center gap-2 text-xl mt-2"
    >
      <a
        class="hover:text-blue-main hover:underline transition-colors duration-200"
        href="https://github.com/tjpetzIII"
      >
        GitHub
      </a>
      <span
        class="text-gray-400"
      >
        |
      </span>
      <a
        class="hover:text-blue-main hover:underline transition-colors duration-200"
        href="https://www.linkedin.com/in/tpetz1/"
      >
        LinkedIn
      </a>
    </nav>
  </div>
</DocumentFragment>
`;
